src/Command.js

```diff
@@ -173,6 +176,43 @@ export class Command {
 		})
 	}
 
+	schedule() {
+		return Promise.reject('Command does not implement schedule(): ' + this.name)
```

Comment:
I’m thinking it makes the most sense to explicitly register scheduled commands in a provider (`CommandProvider` likely).

Two big feature gaps I see:
1. Currently can’t run the same command multiple times (just at different intervals)
2. No way to pass in cli arguments/options

Laravel handles this pretty cleanly:

```php
$schedule->command('emails:send --force')->daily();
$schedule->command(EmailsCommand::class, ['--force'])->daily();
```